reconcile/utils/jump_host.py

```diff
@@ -71,3 +78,25 @@ def get_ssh_base_cmd(self):
             '-o', 'StrictHostKeyChecking=yes',
             '-o', 'UserKnownHostsFile={}'.format(self.known_hosts_file),
             '-i', self.identity_file, '-p', str(self.port), user_host]
+
+    def create_ssh_tunnel(self, local_port, remote_port):
+        key = f'{self.hostname}-{local_port}-{remote_port}'
+        JumpHostSSH.tunnel_lock.acquire()
```

Comment:
use `with`

Reply:
https://docs.python.org/3/library/threading.html#using-locks-conditions-and-semaphores-in-the-with-statement

Reply:
Done